src/ui/qgisapp.ui

```diff
@@ -2673,6 +2675,26 @@ Acts on the currently active layer only.</string>
     <string>Hide Selected Layers</string>
    </property>
   </action>
+  <action name="mActionToggleSelectedLayers">
+   <property name="icon">
+    <iconset resource="../../images/images.qrc">
+     <normaloff>:/images/themes/default/mActionToggleSelectedLayers.svg</normaloff>:/images/themes/default/mActionToggleSelectedLayers.svg</iconset>
+   </property>
+   <property name="text">
+    <string>Toggle Selected Layers</string>
+   </property>
+   <property name="shortcut">
+    <string>Space</string>
+   </property>
```

Comment:
Space should be removed from a global shortcut.

Reply:
This is not a global shortcut, but a widget shortcut (see the next property). I use it only to hive hint to the user that this exists. If have any idea about this, it would be appreciated.

Reply:
This is confusing. You are using an action's shortcut (`mActionToggleSelectedLayers`) to advertise functionality that does not involve the action itself, since the space button is handled directly by capturing the event in `QgsLayerTreeView`. Everywhere else in the ui the shortcut displayed is the global shortcut for the action, we should stick to that.